### DISCUSSION TITLE
Not able to match space character with `consult-line`

### DISCUSSION THREAD

**Discussion by david-alvarez-rosa** [minad/consult#873] [Q&A] [answered]
I'm not able to match space character " " when using `consult-line`.

For instance, given the below code snippet

```cpp
int main() {
  auto my_string{"Emacs is so cool!"};
}
```

I have not way of matching `is so` sequence.

**Comment by minad** [comment, marked as answer]
I assume you use the Orderless completion style? You can set `orderless-component-separator` to `orderless-escapable-split-on-space` and then escape the space with backlash.

**Reply by david-alvarez-rosa** [reply]
Thank you @minad—that works indeed!

**Reply by oantolin** [reply]
Or you can use the Emacs regexp `\s-` to match a space (this would work with orderless's default settings too).

**Reply by david-alvarez-rosa** [reply]
Thanks @oantolin!
